runelite-client/src/main/java/net/runelite/client/plugins/theatre/rooms/nylocas/NyloHandler.java

```diff
@@ -368,6 +393,127 @@ public void onGameTick()
 		}
 	}
 
+	private int checkAttackStyle(String itemName)
+	{
+		switch (itemName.toLowerCase())
+		{
+			case "toxic blowpipe":
+			case "twisted bow":
+			case "craw's bow":
+				return 1;
+			case "abyssal whip":
+			case "abyssal tentacle":
+			case "scythe of vitur":
+			case "scythe of vitur (uncharged)":
+			case "ham joint":
+			case "bandos godsword":
+			case "bandos godsword (or)":
+			case "dragon warhammer":
+			case "dragon claws":
+			case "event rpg":
+			case "ghrazi rapier":
+			case "blade of saeldor":
+			case "crystal halberd":
+			case "dragon scimitar":
+				return 2;
+			case "kodai wand":
+			case "master wand":
+			case "trident of the seas":
+			case "trident of the swamp":
+			case "sanguinesti staff":
+			case "iban's staff":
+			case "iban's staff (u)":
+			case "trident of the swamp (e)":
+			case "trident of the seas (e)":
+				return 3;
+			case "red chinchompa":
+			case "chinchompa":
+			case "black chinchompa":
+			case "armadyl crossbow":
+			case "dragon crossbow":
+			case "rune crossbow":
+				return 4;
+			case "avernic defender":
+			case "dragon defender":
+			case "dragon defender (t)":
+				if (attackStyle == 1)
+				{
+					return 2;
+				}
+				else
+				{
+					return attackStyle;
+				}
+			default:
+				return attackStyle;
+		}
+	}
+
+	public void onMenuOptionClicked(MenuOptionClicked event)
+	{
+		if (!plugin.isNylocasMenuSwap())
+		{
+			attackStyle = 0;
+			removeMenuSwaps();
+			return;
+		}
+		final String menuOption = event.getOption();
+		if (!menuOption.equalsIgnoreCase("equip") && !menuOption.equalsIgnoreCase("attack"))
+		{
+			return;
+		}
+
+		if (attackStyle == 0)
```

Comment:
Why aren't you using a enum or something like that? Even if those're already duplicated 100 times it's better than magic numbers

Reply:
fixed already